include/fastdds/rtps/builtin/BuiltinProtocols.h

```diff
@@ -116,8 +113,8 @@ class BuiltinProtocols
     PDP* mp_PDP;
     //!Pointer to the WLP
     WLP* mp_WLP;
-    //!Pointer to the TypeLookupManager
-    fastdds::dds::builtin::TypeLookupManager* tlm_;
+    //!The TypeLookupManager
+    fastdds::dds::builtin::TypeLookupManager tlm_;
```

Comment:
I suggest using a more explicit name instead of `tlm`, for instance, `typelookup_manager`.

Reply:
Changed in [9cf4645](https://github.com/eProsima/Fast-DDS/pull/4126/commits/9cf46451cbc0cc26f54f25c7979ada4904c22109)